src/common/components/ApprovedByField/ApprovedByField.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+
+import { stripesConnect } from '@folio/stripes/core';
+import { KeyValue } from '@folio/stripes/components';
+
+import { get } from 'lodash';
+
+import { USERS } from '../../resources';
+import { USERS_API } from '../../constants/api';
+
+const ApprovedByField = ({ approvedById, resources }) => {
+  const user = get(resources, ['user', 'records', 0]);
```

Comment:
I think you might miss the case when user is 404, and the previously loaded user will be displayed, what is the bug. Maybe it should look like 
```
const ApprovedBy = ({ resources }) => {
  const { failed, records } = resources.approvedByUser || {};
  const approvedByUser = failed ? null : get(records, '0');

  return (
    <KeyValue
      label={<FormattedMessage id="ui-invoice.invoice.approvedBy" />}
      value={getUserName(approvedByUser)}
    />
  );
};
```

Reply:
fixed. thanx